cdap-watchdog/pom.xml

```diff
@@ -70,6 +70,27 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-common</artifactId>
+      <scope>provided</scope>
```

Comment:
Yes, we use provided scope for all HBase dependencies.  Prior to this it was pulling in HBase as a transitive dependency from elsewhere.